playbooks/roles/airship-prepare/tasks/main.yml

```diff
@@ -15,23 +15,23 @@
 - name: Set socok8s_dcm_vip in extravars
   lineinfile:
     path: "{{ socok8s_extravars }}"
-    regexp: "^socok8s_dcm_vip.*"
     line: "socok8s_dcm_vip: {{ socok8s_ext_vip }}"
+  when: socok8s_dcm_vip is not defined
```

Comment:
I think this is a bad practice.

If this is necessary AND user input is required, then we should document it, and fail if not provided.

If user input is not required, then we should (eventually) document it as an extra feature, and use some kind of default setting.

Reply:
I agree that this isn't the best practice, I believe these were originally added as a way of setting default values so that user input was no longer required, but now that I think about it, we should just move them to a defaults file.